FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Vladimir Jeremic on 4/27/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedFeedCache
+
+	static func feedImages(from localFeedImges: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
+		return NSOrderedSet(array: localFeedImges.map({ feedImage(from: $0, in: context) }))
```

Comment:
There's a typo in `localFeedImges`
Please replace `localFeedImges` with `localFeedImages`

```suggestion
	static func feedImages(from localFeedImages: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
		return NSOrderedSet(array: localFeedImages.map({ feedImage(from: $0, in: context) }))
```